package.json

```diff
@@ -56,8 +56,8 @@
     "typescript": "^4.2.5"
   },
   "peerDependencies": {
-    "firebase-admin": ">=6.0.0",
-    "firebase-functions": ">=3.23.0",
+    "firebase-admin": "^8.0.0 || ^9.0.0 || ^10.0.0 || ^11.0.0",
+    "firebase-functions": "^4.0.0",
```

Comment:
Tagged versions don't get picked up with `^` or `>`.

You might want to update this to include the tag?
```
"firebase-functions":  "^4.0.0 || 4.0.0-rc.0",
```

Reply:
Since we are about to release 4.0.0 anyway, I think it's safe to keep things as is.